pkgs/tools/misc/hoard/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitHub, ncurses, openssl, pkg-config, stdenv, Security, fetchpatch }:
```

```suggestion
{ lib, rustPlatform, fetchFromGitHub, ncurses, openssl, pkg-config, stdenv, Security }:
```
fetchpatch isn't needed